fr.openfisca.org/api/latest/requirements.txt

```diff
@@ -1,3 +1,3 @@
-OpenFisca-France
 OpenFisca-Core[web-api]
+OpenFisca-France
```

Comment:
Changing the order of installation should protect us from what I think what the motivation behind #79 

pip will first install the latest OpenFisca Core and its extra Web API dependencies. 
It will then install the latest OpenFisca-France. If that is not compatible with the latest Core, it will uninstall it and install instead the latest compatible Core version. (It won't reinstall the Web API extra dep, but that doesn't really matter).